mesonbuild/wrap/wrap.py

```diff
@@ -515,6 +517,25 @@ def get_svn(self) -> None:
         subprocess.check_call([svn, 'checkout', '-r', revno, self.wrap.get('url'),
                                self.directory], cwd=self.subdir_root)
 
+    def get_netrc_credentials(self, netloc: str) -> T.Optional[T.Tuple[str, str]]:
+        try:
+            rc = netrc()
```

Comment:
Why not init netrc in the class constructor (like `self.netrc`). To avoid doing `netrc()` in each request.